hmcts/fpl-ccd-configuration
# feat: notify admin when any order issued (FPLA-25 / FPLA-1208)

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPLA-25
https://tools.hmcts.net/jira/browse/FPLA-1208

https://www.notifications.service.gov.uk/services/12f756df-f01d-4a32-a405-e1ea8a494fbb/templates/c652061a-31db-4ec5-bcf7-4acec5943658

### Change description ###
Updated notification handler to send an email to admin when any of the following orders are issued:

- 'Create an order' order (25)
- Notice of placement order (25)
- CMO (1208)

If any representatives need serving by post, then include their addresses in the notification

If it's a CMO, then include grey callout

### Test cases for QA: ###

Case with no representatives: verify notification sent to admin after

- Issuing CMO as judge (should have grey callout)
- Creating an order from 'Create an order'
- Notice of placement order uploaded/changed in 'Placement' event

Case with representatives who need serving by post: verify notification sent to admin after

- Issuing CMO as judge (should have grey callout)
- Creating an order from 'Create an order'
- Notice of placement order uploaded/changed in 'Placement' event. Should send notification for each new/changed notice of placement order uploaded during event

 (should include representatives address in notification)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```
